### ISSUE TITLE
[Enhancement] Spellchecker: do not red underline while writing a word

### ISSUE THREAD

**Issue by BellLongworth** [Zettlr/Zettlr#2315]
### Description

Today the spellchecker "works" while typing. For example, if I'm typing "mistake" I will get the red underline after "mis", "mista", "mistak". This cause a micro-attention loss and hurts the typing flow.

### Proposed Changes

Have spellchecker do its thing only after I typed space and moved to the next word.

### Caveats

Don't see any.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Beta (if applicable)

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Operating System Version

Windows 10

**Comment by nathanlesage** [comment]
I don't think that this is possible, since we're making use of an overlay mode to apply the formatting (red underlines) which is being executed in lockstep with your writing. An alternative would be to use text markers for that which we could delay, however that will interfere with _a lot of other components_ of the editor, making this completely unfeasible.

Furthermore, since you mention it's just a "micro-loss", I am going to close this as "wontfix". Maybe after migration to CodeMirror 6 this could be possible, but we'll have to see. If there is no betterment to your predicament, please make sure to keep this in your mind and open a new issue once we have migrated to CodeMirror 6, maybe then I'll have better news for you!
